### PR TITLE
More CoAL improvements

### PR DESCRIPTION
**Pull request by MadMan310**
### Multiblock Changes
- Added more input bus canidates 
image
- Added separate input bus capability and support for the new GUI buttons by @minecraft7771 
image
- Added more item and fluid input slots to NEI
image

### New Balances
- Energy and Duration is multiplied 64x (The component output multiplier).
image
- This makes the multi less OP. Even though there is a much higher power cost and time sink, it is still more efficient because it uses less items.
- This was based off of the Productivity Modules from Factorio, where they decrease the item inputs required for the same amount of outputs, but also vastly increase the power consumption and time.

### CODE REVIEW

File: src/main/java/goodgenerator/blocks/tileEntity/ComponentAssemblyLine.java [hunk 1]
```diff
@@ -47,9 +53,9 @@ public class ComponentAssemblyLine extends GT_MetaTileEntity_LongPowerUsageBase<
                     STRUCTURE_PIECE_MAIN,
                     new String[][] {
                             { "         ", "   III   ", " HHI~IHH ", "HH III HH", "H       H", "H       H", "H  JJJ  H",
-                                    "H  N N  H", "H  N N  H", "HHHHHHHHH" },
-                            { "         ", " EHHHHHE ", "E       E", "H       H", "A       A", "A       A", "A  HHH  A",
-                                    "A       A", "A       A", "MHHHHHHHM" },
+                                    "H  JJJ  H", "H  N N  H", "HHHHHHHHH" },
+                            { "         ", " EHHHHHE ", "E       E", "H       H", "A       A", "A       A", "A       A",
+                                    "A  HHH  A", "A       A", "MHHHHHHHM" },
```
Comment (GlodBlock) [inline src/main/java/goodgenerator/blocks/tileEntity/ComponentAssemblyLine.java:58]: it will break existed coal setups

Comment (MadMan310) [reply src/main/java/goodgenerator/blocks/tileEntity/ComponentAssemblyLine.java:58]: yes, but the change is very minor. plus, the input hatches that are already in their positions before the update will still keep their same positions. not much change is going to be needed.

File: src/main/java/goodgenerator/blocks/tileEntity/ComponentAssemblyLine.java [hunk 2]
```diff
@@ -30,15 +33,18 @@
...
 import gregtech.api.util.GT_Multiblock_Tooltip_Builder;
 import gregtech.api.util.GT_Recipe;
 import gregtech.api.util.GT_StructureUtility;
+import gregtech.api.util.GT_Utility;
 
 public class ComponentAssemblyLine extends GT_MetaTileEntity_LongPowerUsageBase<ComponentAssemblyLine>
         implements ISurvivalConstructable {
 
     private int casingTier;
+    private boolean separateInputBuses = false;
```
Comment (minecraft7771) [inline src/main/java/goodgenerator/blocks/tileEntity/ComponentAssemblyLine.java:47]: @Dream-Master if we want to include https://github.com/GTNewHorizons/GT5-Unofficial/pull/1767 for next stable, then we shouldn't declare input separation here. The mentioned PR introduces a flag for input separation in the GT5U base class, to use with the GUI buttons.

Comment (MadMan310) [reply src/main/java/goodgenerator/blocks/tileEntity/ComponentAssemblyLine.java]: fixed in [bf973d7](https://github.com/GTNewHorizons/GoodGenerator/pull/136/commits/bf973d7df0f3c8dbf3ef7da9e360fd84445691ad)

**Review by minecraft7771 — APPROVED** [review]

**Pull request merged**